### PR TITLE
docs(web-readme): add docker local deployment guide

### PR DESCRIPTION
**Pull request by DM1-1**
### PR Checklist

If you have any questions, you can refer to the [Contributing Guide](https://github.com/emqx/MQTTX/blob/main/.github/CONTRIBUTING.md)

#### Issue Number

#1575 

#### What is the new behavior?

Add docker local deployment guide. 

#### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

#### Specific Instructions

Support for PR#1619

### CODE REVIEW

File: web/README-CN.md [hunk 1]
```diff
@@ -39,6 +39,10 @@ docker pull emqx/mqttx-web:latest
 docker run -d --name mqttx-web -p 80:80 emqx/mqttx-web:latest
 ```
 
+## 手动编译部署
+
+支持通过 env 修改配置，编译后可以使用 Nginx 或者 Docker 部署 MQTTX Web。
```
Comment (Red-Asuka) [inline web/README-CN.md:44]: 如果你需要进行某些个性化设置，例如改变默认的连接路径、部署路径等，你可以编辑 `.env` 配置文件并手动编译来实现这些修改。如果你想知道如何编译项目和打包 docker 镜像，你可以参考我们的开发部分。

File: web/README.md [hunk 2]
```diff
@@ -42,6 +42,10 @@ docker pull emqx/mqttx-web:latest
 docker run -d --name mqttx-web -p 80:80 emqx/mqttx-web:latest
 ```
 
+## Manual compilation and deployment
+
+Supports modifying configuration through env, and after compilation, you can deploy MQTTX Web using Nginx or Docker.
```
Comment (Red-Asuka) [inline web/README.md:47]: If you need to make some personalized settings, such as changing the default connection path, deployment path, etc., you can achieve these modifications by editing the `.env` configuration file and compiling manually. For information on how to compile the project and package docker images, you can refer to the development section.

File: web/README-CN.md [hunk 3]
```diff
@@ -77,6 +81,15 @@ yarn run build
 
 # 编译并启动一个本地 HTTP 服务器以进行测试
 yarn run start
+
+# 编译和压缩以构建生产版本，用于Docker容器化
```
Comment (Red-Asuka) [inline web/README-CN.md:85]: Spaces are needed between Chinese and English texts.

File: web/README-CN.md [hunk 4]
```diff
@@ -77,6 +81,15 @@ yarn run build
...
+yarn run build:docker
+
+# 构建 docker 镜像
+docker build -t mqttx-web .
+
+# 运行 docker 镜像
```
Comment (Red-Asuka) [inline web/README-CN.md:91]: docker -> Docker

File: web/README.md [hunk 5]
```diff
@@ -42,6 +42,10 @@ docker pull emqx/mqttx-web:latest
 docker run -d --name mqttx-web -p 80:80 emqx/mqttx-web:latest
 ```
 
+## Manual compilation and deployment
+
+If you need to make some personalized settings, such as changing the default connection path, deployment path, etc., you can achieve these modifications by editing the .env configuration file and compiling manually. For information on how to compile the project and package docker images, you can refer to the [development section](Develop).
```
Comment (Red-Asuka) [inline web/README.md:47]: The anchor is incorrect.

**Review by Red-Asuka — CHANGES_REQUESTED** [review]

**Comment by ysfscream** [comment]
@DM1-1 It seems that I haven't seen updates on the instructions for configuration use, but that's fine. We need to update the documentation at https://github.com/emqx/mqttx-site. After the update, I will introduce how to customize configuration packaging in a new blog post about mqttx-web.

**Comment by DM1-1** [comment]
> @DM1-1 It seems that I haven't seen updates on the instructions for configuration use, but that's fine. We need to update the documentation at https://github.com/emqx/mqttx-site. After the update, I will introduce how to customize configuration packaging in a new blog post about mqttx-web.

I agree with your point that this content is indeed more suitable for presentation through an article.

**Comment by ysfscream** [comment]
@DM1-1 YES. But you still need to clearly these configs in the web docs on mqttx.app, I will directly refer to it later. Thank you.

**Review by ysfscream — APPROVED** [review]

**Comment by DM1-1** [comment]
> @DM1-1 YES. But you still need to clearly these configs in the web docs on mqttx.app, I will directly refer to it later. Thank you.
OK. I will complete the web docs during my available time and ensure that it is promptly updated on mqttx.app.

**Review by Red-Asuka — APPROVED** [review]

**Pull request merged**